v3/docs/gs/sysreqs.md

```diff
@@ -24,11 +24,11 @@ be helpful in creating and verifying a balanced configuration.
 ## Operating System
 
 HSE should work with most modern Linux&reg; 64-bit operating system
-distributions.  We have run HSE on the following.
+distributions.  We have run HSE on the following:
 
-* Red Hat&reg; Enterprise Linux&reg; 8 (RHEL 8)
-* Ubuntu&reg; 18.04 and 20.04
-* Fedora&reg; 34 and 35
+* Red Hat&reg; Enterprise Linux&reg; 8 (RHEL 8) and 9 (RHEL 9)
+* Ubuntu&reg; 18.04, 20.04, 22.04
+* Fedora&reg; 35 and 36
 * AlmaLinux OS 8
```

Comment:
I don't think it is worth mentioning Almalinux here since it is binary compatible with rhel

Reply:
Fair point